Testnet/MintableToken/MintableToken/MintableToken.cs

```diff
@@ -221,6 +235,28 @@ public void MintWithMetadata(Address account, UInt256 amount, string metadata)
         Log(new MintMetadata() { To = account, Amount = amount, Metadata = metadata });
     }
 
+    /// <inheritdoc />
+    public void MintWithMetadataForNetwork(Address account, UInt256 amount, string metadata, string destinationAccount, string destinationNetwork)
```

Comment:
`destinationAccount` should be Address type if it is possible.

Reply:
The choice of string was made to avoid the assumption that the address would always be representable as 20 bytes.